### ISSUE TITLE
Increased 17 hydroxypregnenolone (17OHP)

### ISSUE THREAD

**Issue by e-lbtn** [obophenotype/human-phenotype-ontology#6883]
**Preferred term label:**
Increased 17 hydroxypregnenolone (17OHP)

**Definition (free text, please give PubMed ID)**
Increased of circulating 17 hydroxypregnolone level
(20301592[PMID]_3312397[PMID])

**Parent term (use hpo.jax.org/app)**
Increased circulating androgen level (HP:0030348)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Congenital adrenal hyperplasia due to cytochrome P450 oxidoreductase deficiency (ORPHA 95699)

**Comment by pnrobinson** [comment]
Definition: Increased concentration of 17alpha-hydroxypregnenolone in the blood circulation. 17alpha-hydroxypregnenolone is a 21-carbon steroid that is converted from pregnenolone by steroid 17-alpha-hydroxylase, as an intermediate in the  biosynthesis of gonadal steroid hormones and adrenal corticosteroids.
Comment: Increased concentration of 17 hydroxypregnolone level can be observed in adrenal hyperplasia, congenital, due to 3-beta-hydroxysteroid dehydrogenase 2 deficiency.
